dashboard/modules/snapshot/snapshot_head.py

```diff
@@ -97,9 +97,9 @@ async def get_serve_info(self):
         # TODO(architkulkarni): Use _internal_kv_list to get all Serve
         # controllers.  Currently we only get the detached one.  Non-detached
         # ones have name = SERVE_CONTROLLER_NAME + random letters.
-        key = format_key(SERVE_CONTROLLER_NAME, SERVE_SNAPSHOT_KEY)
-
-        return json.loads(_internal_kv_get(key, client) or "{}")
+        key = get_storage_key(SERVE_CONTROLLER_NAME, SERVE_SNAPSHOT_KEY)
+        val_bytes = _internal_kv_get(key, client) or "[]".encode("utf-8")
```

Comment:
```suggestion
        val_bytes = _internal_kv_get(key, client) or "{}".encode("utf-8")
```
?

Reply:
ah good call, it's empty dict from its source in controller